client/src/components/Typography/Heading.js

```diff
@@ -33,6 +33,10 @@ const Heading = styled(Title)`
   }
   &.h4 {
     font-size: ${heading.four};
+    @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
```

Comment:
```suggestion
```

Thanks @mel-chiu this is good but I think instead of adding the mobile `margin-top: 10%` rule here as it is used in a few other places and might have some unintended consequences.

We could add a styled component that extends this mobile margin-top: 10% rule ont he Login.js page only.

Also why the font-size change?

Otherwise this is great.